modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/idp/v1/model/FederatedAuthenticatorRequest.java

```diff
@@ -234,16 +271,63 @@ public FederatedAuthenticator addProperty(Property property) {
             return this;
         }
 
+        /**
+         *
+         **/
+        public FederatedAuthenticator definedBy(DefinedByEnum definedBy) {
+
+            this.definedBy = definedBy;
+            return this;
+        }
+
+        @ApiModelProperty(value = "")
+        @JsonProperty("definedBy")
+        @Valid
+        public DefinedByEnum getDefinedBy() {
+            return definedBy;
+        }
+        public void setDefinedBy(DefinedByEnum definedBy) {
+            this.definedBy = definedBy;
+        }
+
+        /**
+         **/
+        public FederatedAuthenticator endpoint(Endpoint endpoint) {
+
+            this.endpoint = endpoint;
+            return this;
+        }
+
+        @ApiModelProperty(value = "")
+        @JsonProperty("endpoint")
+        @Valid
+        public Endpoint getEndpoint() {
+            return endpoint;
+        }
+        public void setEndpoint(Endpoint endpoint) {
+            this.endpoint = endpoint;
+        }
+
```

Comment:
check formatting

Reply:
These file are auto generated files from the API server repo (from gen component).